Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -33,6 +33,29 @@ class AppStoreConnectService {
         try InviteTesterOperation(options: options).execute(with: requestor)
     }
 
+    func getBetaTesterInfo(
+        email: String,
+        limitApps: Int?,
+        limitBuilds: Int?,
+        limitBetaGroups: Int?
+    ) throws -> BetaTester {
+        let operation = GetBetaTestersInfoOperation(
+            options: .init(
+                email: email,
+                limitApps: limitApps,
+                limitBuilds: limitBuilds,
+                limitBetaGroups: limitBetaGroups
+            )
+        )
+
+        let betaTesterResponse = try operation.execute(with: requestor).await()
+
+        return betaTesterResponse
+            .data
+            .map { BetaTester($0, betaTesterResponse.included) }
+            .first!
```

Comment:
🤔 I think I was confused here. If we're passing one email address into this operation we should only be getting one beta tester right?

What operation are you using for listing beta testers?
Or is that to be done in a later PR

`.first!` doesn't feel great here